src/components/CurationPage/CollectionRow/CollectionRow.tsx

```diff
@@ -14,6 +14,11 @@ import { Props } from './CollectionRow.types'
 import './CollectionRow.css'
 
 export default class CollectionRow extends React.PureComponent<Props> {
+  componentDidMount = () => {
+    const { collection, onFetchCollectionItems } = this.props
+    onFetchCollectionItems(collection.id)
```

Comment:
What if we load the items for the collection once? We can have a selector that checks if there are items for that collection and avoid the loading time for the curations.